services/database/program.ts

```diff
@@ -23,4 +23,39 @@ async function getPrograms(): Promise<Program[]> {
     return programs;
 }
 
-export { getProgram, getPrograms };
+/**
+ * NOTE: https://www.prisma.io/docs/concepts/components/prisma-client/advanced-type-safety/operating-against-partial-structures-of-model-types
+ * createProgram takes in newProgramData and returns the newly created program
+ * @param newProgramData - data corresponding to new program
+ * @returns Promise<Program> - Promise with the newly created program
+ */
+async function createProgram(
+    newProgramData: CreateProgramInput,
+): Promise<Program> {
+    const program = await prisma.program.create({
+        data: {
+            price: newProgramData.price,
+            online_format: newProgramData.onlineFormat,
+            tag: newProgramData.tag,
+            start_date: newProgramData.startDate,
+            end_date: newProgramData.endDate,
+            is_archived: newProgramData.isArchived,
+        },
+    });
+    return program;
+}
+
+/**
+ * deleteProgram takes in the id of the program and returns the deleted program
+ * @param id - programId of the program to delete
+ * @returns Promise<Program> - Promise with the deleted program
+ */
+async function deleteProgram(id: string): Promise<Program> {
```

Comment:
@feng-eric - are we okay with doing a hard delete? another option would be is to add an `is_deleted` column and set that to true when we delete to preserve the records.

Reply:
I'm okay with hard delete for now. Let's discuss as a team if we see an use case for soft delete. We can refactor this in the future to accommodate for soft delete if we want it.